src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -32,12 +32,23 @@ function LtiConfigForm({
   const isInvalidConsumerSecret = touched.consumerSecret && errors.consumerSecret;
   const isInvalidLaunchUrl = touched.launchUrl && errors.launchUrl;
 
+  const handleValidation = (event) => {
+    event.preventDefault();
+    validateForm().then(() => {
+      handleConfigError(true);
+      handleSubmit();
+    });
+  };
+
+  useEffect(() => {
+    handleConfigError(Object.keys(errors).length > 0);
+  }, [errors]);
+
   return (
-    <Card className="mb-5 pt-3 px-5 pb-5" data-testid="ltiConfigForm">
-      <Form ref={formRef} onSubmit={handleSubmit}>
-        <h3>{title}</h3>
-        <AppConfigFormDivider />
-        <p>
+    <Card className="mb-5 p-4" data-testid="ltiConfigForm">
+      <Form ref={formRef} onSubmit={handleValidation}>
```

Comment:
it sounds like form is only being validated on submit.
what about `vlidateAndSubmit` or anything similar?